apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue

```diff
@@ -26,7 +26,7 @@
                     {{ props.label }}
                 </p>
                 <div class="label-description">
-                    <div v-if="state.isScheduleActivated"
+                    <div v-if="state.isScheduleActivated && props.item.schedule.hours?.length > 0"
```

Comment:
issue: optional chaining does not work with vue2 on template

Reply:
oops!!!!!!!!!!!!, right! thx!!